aea/manager/project.py

```diff
@@ -131,12 +131,17 @@ class AgentAlias(_Base):
     """Agent alias representation."""
 
     def __init__(
-        self, project: Project, agent_name: str, keys_dir: Optional[str],
-    ) -> None:
+        self,
+        project: Project,
+        agent_name: str,
+        keys_dir: Optional[str],
+        data_dir: Optional[str],
+    ):
         """Init agent alias with project, config, name, agent, builder."""
         self.project = project
         self.agent_name = agent_name
         self._keys_dir = keys_dir or project.path
+        self._data_dir = data_dir or project.path
```

Comment:
This "or" is a problem. Why would we have it? I think the dirs should be required arguments

Reply:
Also, clearly the Dara dir of the agent alias cannot be the data dir of the manager. It needs to be `data/agent_name` or something similar.

Also, why do we still have a keys dir?